README.md

```diff
@@ -48,20 +44,36 @@ by running it with the `--help` flag or reading the [rustc dev guide][rustcguide
 [gettingstarted]: https://rustc-dev-guide.rust-lang.org/getting-started.html
 [rustcguidebuild]: https://rustc-dev-guide.rust-lang.org/building/how-to-build-and-run.html
 
-### Building on a Unix-like system
-1. Make sure you have installed the dependencies:
-
-   * `g++` 5.1 or later or `clang++` 3.5 or later
-   * `python` 3 or 2.7
-   * GNU `make` 3.81 or later
-   * `cmake` 3.13.4 or later
-   * `ninja`
-   * `curl`
+### Dependencies
+
+Make sure you have installed the dependencies:
+
+   * `python` 3 or 2.7 (under the name python; python2 or python3 will not work)
```

Comment:
Is it really true that python2 or python3 won't work? What creates that requirement?

Reply:
`./x.py` requires exactly `python`. Running `python3 x.py` still works as usual. How about suggesting `python x.py` in the readme and removing the parentheses about python2/python3?